📝 Update README.md (#822)
ajnart/homarr: README.md

- 📄 Detailed documentation and active community
- 🔍 Search through the web or supported integrations in an instant
- 🏴󠁧󠁢󠁮󠁩󠁲󠁿 Monitor your application with a built-in status system

> - 🦞 Comprehensive built-in icon picker with over 7'000 icons

Reviewer: ```suggestion
- 🦞 Comprehensive built-in icon picker with over 7000 icons
```

> - 🐳 Easy deployment with Docker, unRAID and Synology

Reviewer: ```suggestion
- 🐳 Easy deployment with Docker, unRAID, and Synology
```

Author: grammar?

Reviewer: What do you mean grammar?

> - 🚀 Compatible with any major consumer hardware (x86, Raspberry Pi, old Laptops, ...)

Reviewer: ```suggestion
- 🚀 Compatible with any major consumer hardware (x86, Raspberry Pi, old laptops, ...)
```

![Installation Section](docs/section-installation.png)

> Since we are developing Homarr very frequently, we recommend reading our official installation guides:

Reviewer: ```suggestion
Since we are updating Homarr very frequently, we recommend reading our official installation guides:
```

  <img src="https://cdn.ko-fi.com/cdn/kofi3.png?v=3" width="200" />
</a>

> You can also support us [translating the entire project in as many languages as possible](https://homarr.dev/docs/community/translations), contribute directly to the code or documentation.

Reviewer: ```suggestion
You can also support us by helping with [translating the entire project](https://homarr.dev/docs/community/translations)  to as many language as possible or contributing directly to the code or documentation.
```
